### PR TITLE
Update Helm chart release dialogs

### PR DESCRIPTION
**Pull request by paigecalvert**
Updates to match the new vendor portal UI flow for adding a Helm chart to a Helm only release:
* Quick start workflow updated to reflect the current dialogs in the UI: https://deploy-preview-2098--replicated-docs.netlify.app/vendor/replicated-onboarding#workflow
* Creating releases with the vendor portal updated to reflect the current dialogs in the UI: https://deploy-preview-2098--replicated-docs.netlify.app/vendor/releases-creating-releases#create-a-release
* Update command in Packaging a Helm Chart for a Release to use the `-u` flag, use `PATH_TO_CHART` placeholder text, and also explain how -u is the same as --dependency-update: https://deploy-preview-2098--replicated-docs.netlify.app/vendor/helm-install-release 

Also, made some unrelated updates to creating releases with the CLI to use more precise language and clean up extra fluff: https://deploy-preview-2098--replicated-docs.netlify.app/vendor/releases-creating-cli

Did _not_ update the [Helm/KOTS getting started tutorial](https://deploy-preview-2098--replicated-docs.netlify.app/vendor/tutorial-kots-helm-package-chart) since the steps for packaging the chart and adding it to the release are a) within the context of a step by step tutorial so they should just be able to copy and paste all the commands exactly as shown, and b) done in the CLI/not the UI

### CODE REVIEW

File: docs/partials/helm/_helm-package.mdx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+```bash
+helm package -u PATH_TO_CHART
+```  
+Where:
+* `-u` or `--dependency-update` is an option for the `helm package` command that updates chart dependencies before packaging. For more information, see [Helm Package](https://helm.sh/docs/helm/helm_package/) in the Helm documentation.
+* `PATH_TO_CHART` is the path to the Helm chart in your local directory. For example, `helm package -u .`.
+
+The Helm chart, including any dependencies, is packaged and copied to your current directory in a `.tgz` file. The file uses the naming convention: `CHART_NAME-VERSION.tgz`. For example, `postgresql-8.1.2.tgz`.
```
Comment (paigecalvert) [inline docs/partials/helm/_helm-package.mdx:8]: ^ Added this packaging step to a partial to make sure it's the same wherever it's used

File: docs/partials/releases/_kots-helm-release-promotion-xref.mdx
Comment (paigecalvert) [inline docs/partials/releases/_kots-helm-release-promotion-xref.mdx:1]: Unrelated change to remove an unnecessary partial for the creating releases topic

File: docs/vendor/helm-install-release.md [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
+import HelmPackage from "../partials/helm/_helm-package.mdx"
 
 # Packaging a Helm Chart for a Release
 
 This topic describes how to package a Helm chart and the Replicated SDK into a chart archive that can be added to a release.
 
 ## Overview
 
-To add a Helm chart to a release, you first add the Replicated SDK as a dependency of the Helm chart and then package the chart and its dependencies into a `.tgz` chart archive.
+To add a Helm chart to a release, you first add the Replicated SDK as a dependency of the Helm chart and then package the chart and its dependencies as a `.tgz` chart archive.
```
Comment (paigecalvert) [inline docs/vendor/helm-install-release.md:11]: ^ changed language to "as a tgz" everywhere so that it's consistent throughout the docs and also matches the text in the UI

File: docs/vendor/helm-install-release.md [hunk 3]
```diff
@@ -31,20 +32,10 @@ To package a Helm chart so that it can be added to a release:
...
 
-1. Update the `charts/` directory:
+1. Update dependencies and package the chart as a `.tgz` file:
 
-   ```
-   helm dependency update
-   ```
-   
-1. Package the Helm chart into a `.tgz` archive:
+    <HelmPackage/>
```
Comment (paigecalvert) [inline docs/vendor/helm-install-release.md:37]: ^ replaced the step in Packaging a Helm Chart for a Release with the new partial

File: docs/vendor/releases-creating-cli.mdx [hunk 4]
```diff
@@ -24,129 +24,113 @@ Before you create a release using the replicated CLI, complete the following pre
 
 ## Create a Release From a Local Directory {#dir}
 
-You can use the replicated CLI to create a release from the local directory that contains your application files.
+You can use the replicated CLI to create a release from a local directory that contains the release files.
```
Comment (paigecalvert) [inline docs/vendor/releases-creating-cli.mdx:27]: Did some general language simplification and cleanup throughout this procedure

File: docs/vendor/releases-creating-releases.mdx [hunk 5]
```diff
@@ -30,18 +23,18 @@ To create and promote a release in the vendor portal:
...
 1. If a drop down is displayed, select one of the following options:
    * **Create Helm-only release**: Create a release with one or more Helm charts. Supports installations with the Helm CLI only.
-   * **Create KOTS release**: Create a release with one or more Helm charts or standard Kubernetes manifests. Supports installations with Replicated KOTS. For Helm charts, supports installation with KOTS and installation with the Helm CLI in the same release.
+   * **Create KOTS release**: Create a release that supports installations with Replicated KOTS. KOTS releases can contain Kubernetes manifests and one or more Helm charts. KOTS releases that contain Helm charts can also be installed with the Helm CLI.
 
    If no drop down is displayed, continue to the next step.  
 
 1. Add your files to the release:
-   * For Helm-only releases, in the **Upload Helm Chart** dialog, drag and drop or select a Helm chart `.tgz` file.
+   * **Helm-only**: For Helm-only releases, follow the steps in the **Add the Replicated SDK to your Helm Chart**  dialog to package your chart and upload a `.tgz` chart archive.
```
Comment (paigecalvert) [inline docs/vendor/releases-creating-releases.mdx:31]: ^ Updated this step to reflect the current workflow and update the screenshot to the new dialog

File: docs/vendor/replicated-onboarding.md [hunk 6]
```diff
@@ -83,37 +84,41 @@ To begin onboarding to the Replicated platform with a Helm chart:
...
+      :::
+   
+   1. In the **Add the Replicated SDK to your Helm chart** dialog, click **Yes, I have a Helm chart for my app that I'd like to use**, and then click **Next**.
 
-   The Replicated SDK is a Helm chart that provides access to Replicated features and can be installed as a small service alongside your application. For more information, see [About the Replicated SDK (Beta)](/vendor/replicated-sdk-overview).
+      <img alt="Add the SDK dialog" src="/images/do-you-have-a-helm-chart-modal.png" width="500px"/>
+      [View a larger version of this image](/images/do-you-have-a-helm-chart-modal.png)
 
-1. Update dependencies then package the Helm chart to a `.tgz` file:
+   1. Follow the steps in the dialog to add the Replicated SDK, package the chart to a `.tgz`, and then upload the `.tgz`:
```
Comment (paigecalvert) [inline docs/vendor/replicated-onboarding.md:100]: Updated this step with the current workflow in the UI and with the new screenshot

File: docs/vendor/replicated-onboarding.md [hunk 7]
```diff
@@ -83,37 +84,41 @@ To begin onboarding to the Replicated platform with a Helm chart:
...
 
-   ```bash
-   helm package . --dependency-update
-   ```
+      <img alt="Upload Helm chart dialog" src="/images/upload-helm-chart-modal.png" width="500px"/>
+      [View a larger version of this image](/images/upload-helm-chart-modal.png)   
 
-   For more information, see [Packaging a Helm Chart for a Release](helm-install-release).  
+       The following describes the steps in the dialog:
```
Comment (paigecalvert) [inline docs/vendor/replicated-onboarding.md:105]: Reiterated the steps in the dialog in this quick start procedure since it's supposedly the first time the user is walking through these steps and so I wanted to provide a little extra explanation for each step

File: docs/vendor/releases-creating-cli.mdx [hunk 8]
```diff
@@ -24,129 +24,113 @@ Before you create a release using the replicated CLI, complete the following pre
...
-   1. For each Helm chart, package the chart and its dependencies into `.tgz` files:
+   1. Package each Helm chart as a `.tgz` file. See [Packaging a Helm Chart for a Release](/vendor/helm-install-release).
 
-      ```
-      helm package . --dependency-update
-      ```
+   1. Move the `.tgz` file or files to the local directory that contains the release files:
 
-      For more information, see [Helm Package](https://helm.sh/docs/helm/helm_package/) in the Helm documentation.
+       ```
```
Comment (brianrhea) [inline docs/vendor/releases-creating-cli.mdx:37]:
```suggestion
       ```bash
```

File: docs/vendor/releases-creating-cli.mdx [hunk 9]
```diff
@@ -24,129 +24,113 @@ Before you create a release using the replicated CLI, complete the following pre
...
-   1. Move the `.tgz` file or files to the local directory with your application files:
+       **Example**
 
-      ```
-      mv CHART_TGZ PATH_TO_APP_DIR
-      ```
-      Replace:
-      * `CHART_TGZ` with the `.tgz` Helm chart package.
-      * `PATH_TO_APP_DIR` with path to the local directory with your application files.
+       ```
```
Comment (brianrhea) [inline docs/vendor/releases-creating-cli.mdx:46]:
```suggestion
       ```bash
```
